src/backend/t/0015-BSSched-EventSource-Directory.t

```diff
@@ -257,7 +257,9 @@ push(@files_to_remove,"$ev_dir/$file_name");
 # CLEANUP
 unshift(@files_to_remove,"$eventdir_base/$arch/$evname");
 
-map { ( -d $_ ) ? rmdir $_ : unlink $_ ; die "$_: $!" if $!;} @files_to_remove;
+eval {
+  map { ( -d $_ ) ? rmdir $_ : unlink $_ ; die "$_: $!" if $!;} @files_to_remove;
```

Comment:
Why not just remove the "die" or change into a "warn" instead of "eval" the whole thing ?